tests/test_regression.py

```diff
@@ -485,6 +485,17 @@ def test_30_remove(self):
 
         expected = ['5.150.145.0/24', '109.161.64.0/20']
         self.assertEqual(expected, [n.prefix for n in tree])
+		
+	def test_31_covering_with_delete_error(self):
+        tree = radix.Radix()
+        tree.add('91.187.124.0/24')
+        tree.add('91.187.125.0/24')
+        tree.add('91.187.124.0/23')
+        tree.delete('91.187.124.0/23')
+        tree.add('91.187.124.0/23')
```

Comment:
Yes, so in this change set, with this test as-is, it seems like having the setup include the sequence of:

```
tree.add('91.187.124.0/23')
tree.delete('91.187.124.0/23')
tree.add('91.187.124.0/23')
```

Will result in `91.187.124.0/23` being in the tree, however, below the expected tree only contains `['91.187.124.0/24', '91.187.125.0/24']` which seems incorrect behavior since the last thing we did we add `91.187.124.0/23`.

Other than that (and an indentation issue) I think this is good to go.